raster-foundry/raster-foundry
# Remove SceneToProjectDao and deal with fallout

## Overview

This PR removes the SceneToProjectDao (and a lot of other things) and handles the fallout. Most of the work to facilitate this was done already in previous work to make project requests use default layers, so there's not that much interesting going on here.

### Checklist


## Notes

~Still need to make sure `batch` and `backsplash-export` subprojects are fine, but server interaction is done~

~Just missing migration to remove table and add an on delete cascade somewhere else -- tomorrow morning~

Skipping the migration to drop the table out of fear -- it's a trivial change and I'd rather leave it, since I don't know how we'll get it back if we realize we actually needed it for something even in staging.

## Testing Instructions

 * wander around project scene routes
 * you shouldn't notice anything amiss
 * view an existing analysis
 * it should still be fine

Closes #4516
